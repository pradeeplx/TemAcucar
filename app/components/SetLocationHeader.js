import React from 'react'
import { View, Platform, StyleSheet } from 'react-native'
import Colors from "../Colors"
import Headline from "./Headline"

export default SetLocationHeader = () => (
  <View style={{
    borderColor: Colors.pink,
    borderBottomWidth: StyleSheet.hairlineWidth,
    alignSelf: 'stretch',
  }}>
    <Headline style={{
      marginTop: (Platform.OS === 'ios' ? 10 : 5),
      marginBottom: 5,
    }}>
      Onde você mora?
    </Headline>
  </View>
)
