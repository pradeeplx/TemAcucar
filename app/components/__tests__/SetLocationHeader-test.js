import 'react-native'
import React from 'react'
import SetLocationHeader from '../SetLocationHeader'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(
    <SetLocationHeader />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
