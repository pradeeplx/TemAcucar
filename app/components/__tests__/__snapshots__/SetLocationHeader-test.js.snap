exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "alignSelf": "stretch",
      "borderBottomWidth": 0.5,
      "borderColor": "#FF7F85",
    }
  }>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": "#654C50",
          "fontFamily": "Avenir",
          "fontSize": 18,
        },
        Array [
          Object {
            "color": "#CECED2",
            "fontSize": 24,
            "marginBottom": 20,
            "marginHorizontal": 20,
            "textAlign": "center",
          },
          Object {
            "fontFamily": "Avenir-Medium",
          },
          Object {
            "marginBottom": 5,
            "marginTop": 10,
          },
        ],
      ]
    }>
    Onde vocÃª mora?
  </Text>
</View>
`;
